dev-docs/RFCs/v4.0.0/terms_for_concepts_and_datatypes.md

```diff
@@ -31,8 +33,28 @@ Reaching a consensus on the precise, technical language used when discussing H3
 
 ### Notes
 
-We concede that "hexagon" will still be used informally to refer to the concept of "cell" (As the core team, we do it ourselves!). This should be expected in casual, informal discussions of H3. When **precision** is required, however, we advise the use of strict technical terms like "index", "cell", "hexagon", "pentagon", etc.
+We realize that "hexagon" will still be used informally to refer to the concept of "cell" (As the development team, we do it ourselves!). This should be expected in casual, informal discussions of H3. When **precision** is required, however, we advise the use of strict technical terms like "index", "cell", "hexagon", "pentagon", etc.
 
 ## Use of "Topological"
 
-todo: define "topological"
+We should clarify in the documentation that H3 works with **topological** hexagons and pentagons, rather than true **geometric** hexagons and pentagons.
+
+See, for example: https://github.com/uber/h3-js/issues/53
+
+## Names for H3 Edge Types
+
+Instead of `UnidirectionalEdge`, use the term `DirectedEdge` or `DiEdge`.
+
+For a future undirected edge mode, use the term `UndirectedEdge` or `Edge`.
+
+Function name changes:
+
+|                  Current name                 |          Proposed name           |
+|-----------------------------------------------|----------------------------------|
+| `getH3UnidirectionalEdge`                     | `getH3DiEdge`                    |
+| `h3UnidirectionalEdgeIsValid`                 | `h3DiEdgeIsValid`                |
+| `getOriginH3IndexFromUnidirectionalEdge`      | `getOriginH3CellFromDiEdge`      |
+| `getDestinationH3IndexFromUnidirectionalEdge` | `getDestinationH3CellFromDiEdge` |
+| `getH3IndexesFromUnidirectionalEdge`          | `getH3CellsFromDiEdge`           |
+| `getH3UnidirectionalEdgesFromHexagon`         | `getH3DiEdgesFromCell`           |
```

Comment:
Can we put some documentation above in the `Concepts` section on what a `Cell` is versus an `Index`?

Index is any 64-bit integer representing an H3 object, Cells are hexagons or pentagons, and DirectedEdges are traversals from an origin cell to an adjacent destination cell.

Reply:
How's it look now?